test/container/1.6.0/Dockerfile.dlc.gpu

```diff
@@ -0,0 +1,28 @@
+ARG region
+from 763104351884.dkr.ecr.$region.amazonaws.com/pytorch-training:1.6.0-gpu-py36-cu110-ubuntu16.04
+
+# TODO: Remove once the 1.6.0-gpu-py3 DLC image installs mpi4py
+RUN pip3 install mpi4py==3.0.3
+
+# TODO: Remove once the 1.6.0-gpu-py3 DLC image fixes OpenSSH config
+# Configure OpenSSH so that nodes can communicate with each other
+RUN mkdir -p /var/run/sshd && \
+ sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd
+
+RUN rm -rf /root/.ssh/ && \
+ mkdir -p /root/.ssh/ && \
+ ssh-keygen -q -t rsa -N '' -f /root/.ssh/id_rsa && \
+ cp /root/.ssh/id_rsa.pub /root/.ssh/authorized_keys \
+ && printf "Host *\n StrictHostKeyChecking no\n" >> /root/.ssh/config
```

Comment:
https://github.com/aws/PRIVATE-deep-learning-containers/blob/4d4307b368dc10708556e73face8cd4efa2da4dd/pytorch/training/docker/1.6.0/py3/cu110/Dockerfile.gpu#L294-L298

Similarly we don't need this for cu11

Reply:
See: https://github.com/aws/sagemaker-pytorch-training-toolkit/pull/225#discussion_r540585149